app/src/main/java/org/mozilla/fenix/settings/sitepermissions/SitePermissionsManagePhoneFeatureFragment.kt

```diff
@@ -102,14 +103,9 @@ class SitePermissionsManagePhoneFeatureFragment : Fragment() {
             append(recommendedSpannable)
             this
         }
-        val expectedAction = if (phoneFeature == PhoneFeature.AUTOPLAY) BLOCKED else ASK_TO_ALLOW
+        val expectedAction = if (phoneFeature == PhoneFeature.AUTOPLAY_AUDIBLE) BLOCKED else ASK_TO_ALLOW
```

Comment:
@Baron-Severin should the `else` return `ALLOWED`  ?

Reply:
Spoke about this, the code is a little confusing but working as intended.

Reply:
👍